Cargo.toml

```diff
@@ -22,6 +22,9 @@ bindgen = { optional = true, version = "0.30.0" }
 pkg-config = { optional = true, version = "0.3" }
 cmake = {optional = true, version = "0.1.2"}
 
+[target.'cfg(windows)'.build-dependencies]
```

Comment:
Can you put the use of the `cc` crate behind a feature? On Windows, we could enable the feature by default. Users of other OSs may also want to build Capstone with the `cc` crate.

Reply:
Unfortunately `cargo` currently does not allow default features to vary based on the platform. This means `cc` needs to be a non-option build-dependency. Do you have a better idea?

Reply:
Good point. You could make a `build_capstone_cc` feature that depends on the `cc` crate.
On Windows, the `cc` crate will always be a dependency. However, on other platforms, users can still opt into the feature.

Example:
https://github.com/tmfink/capstone-sys/commit/9f80d6020e252f4f3228894d09c1e4258836954d